bigquery_storage/CHANGELOG.md

```diff
@@ -4,6 +4,24 @@
 
 [1]: https://pypi.org/project/google-cloud-bigquery-storage/#history
 
+## 0.5.0
+
+05-20-2019 09:23 PDT
+
+### Implementation Changes
+
+- Increase default deadline on ReadRows. ([#8030](https://github.com/googleapis/google-cloud-python/pull/8030))
+- Respect timeout on `client.read_rows`. Don't resume on `DEADLINE_EXCEEDED` errors. ([#8025](https://github.com/googleapis/google-cloud-python/pull/8025))
+
+### Documentation
+
+- Use alabaster theme everwhere. ([#8021](https://github.com/googleapis/google-cloud-python/pull/8021))
+
+### Internal / Testing Changes
+
+- Never retry on `DEADLINE_EXCEEDED` error codes. Respect user deadlines. Retry would never succeed, anyway. ([#7879](https://github.com/googleapis/google-cloud-python/pull/7879))
+- Revert [#7879](https://github.com/googleapis/google-cloud-python/pull/7879). Retry `DEADLINE_EXCEEDED` since some client methods set a short deadline at first to reduce tail latency. ([#7921](https://github.com/googleapis/google-cloud-python/pull/7921))
```

Comment:
Let's just drop these two, since they cancel out.

Reply:
Done in 319e8acf214872a52912153c01dfc8d71e13188a